### PR TITLE
[OSDOCS-9233]:Red Hat OpenShift Service on AWS Images book typo

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.14+
Issue:

https://issues.redhat.com/browse/OSDOCS-9233
Link to docs preview:

https://69881--ocpdocs-pr.netlify.app/openshift-rosa/latest/openshift_images/configuring-samples-operator#samples-operator-overview_configuring-samples-operator
QE review:
- [x] QE has approved this change.

Additional information:

image

### CODE REVIEW

**Comment by mletalie** [comment]
Hello @xueli181114, 
   Could you please provide QE review when you get a moment? It looks like the pertinent information was present, it was just conditioned wrong. Thanks.

File: modules/samples-operator-overview.adoc [hunk 1]
```diff
@@ -64,7 +64,7 @@ For {product-title}, the default image registry is
 ifdef::openshift-enterprise[]
 `registry.redhat.io`.
 endif::[]
-ifdef::openshift-origin[]
+ifdef::openshift-rosa,openshift-dedicated[]
```
Comment (bscott-rh) [inline modules/samples-operator-overview.adoc:67]:
```suggestion
ifdef::openshift-rosa,openshift-dedicated,openshift-origin[]
```
if we remove openshift-origin from this conditional, the OKD docs will have an incomplete sentence here. I think it should be re-added

**Review by jeana-redhat — COMMENTED** [review]
LGTM. Merges are currently frozen, but this good to go when that's lifted.

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Comment by mletalie** [comment]
> LGTM. Merges are currently frozen, but this good to go when that's lifted.
> 
> /remove-label merge-review-in-progress /remove-label merge-review-needed

Thanks for the heads up, @jeana-redhat!

**Pull request merged**